i18n.php

```diff
@@ -684,19 +700,19 @@
   ),
   'ka' => 
   array (
-    'and' => 'და<|*',
+    'and' => 'ასევე|და|*',
     'background' => 'კონტექსტი',
-    'but' => 'მაგ­რამ<|*',
+    'but' => 'მაგრამ|თუმცა|*',
     'examples' => 'მაგალითები',
-    'feature' => 'თვისება',
-    'given' => 'მოცემული<|*',
+    'feature' => 'მოთხოვნა|თვისება',
+    'given' => 'მოცემულია|მოცემული|ვთქვათ|*',
     'name' => 'Georgian',
-    'native' => 'ქართველი',
-    'rule' => 'Rule',
-    'scenario' => 'მაგალითად|სცენარის',
-    'scenario_outline' => 'სცენარის ნიმუში',
-    'then' => 'მაშინ<|*',
-    'when' => 'როდესაც<|*',
+    'native' => 'ქართული',
```

Comment:
Translation of `native` has changed, but we don't use it (it is the "native" name of the language e.g. "español" rather than "Spanish")